### PR TITLE
update rules page w/ link to regex tutorial

### PR DESCRIPTION
**Pull request by wdpk**
many people may not know what 'regex' is. now matches filters page. not sure why i didn't update both places when i got the review last time, sorry!

### CODE REVIEW

**Review by RubenOlsen — APPROVED** [review]

File: docs/budgeting/rules/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ While ranking works for the most part, you might want to say "this rule _always_
 
 - `is`/`is not` matches exactly
 - `contains`/`does not contain` matches a substring
-- `matches` is regex
+- `matches` is *[regular expressions](https://regextutorial.org/)*
```
Comment (MatissJanis) [inline docs/budgeting/rules/index.md:29]:
```suggestion
- `matches` are [regular expressions](https://regextutorial.org/)
```

Comment (psybers) [reply docs/budgeting/rules/index.md:29]: To match te first two, perhaps instead make it singular:

```suggestion
- `matches` is a [regular expressions](https://regextutorial.org/)
```

Comment (MatissJanis) [reply docs/budgeting/rules/index.md:29]: Good suggestion. Lets go with @psybers version!

Comment (wdpk) [reply docs/budgeting/rules/index.md]: committed the psybers version!

**Review by MatissJanis — APPROVED** [review]
Thanks!

**Pull request merged**